schema-engine/core/src/state.rs

```diff
@@ -18,7 +18,7 @@ use tracing_futures::Instrument;
 /// `connectors`. Each connector has its own async task, and communicates with the core through
 /// channels. That ensures that each connector is handling requests one at a time to avoid
 /// synchronization issues. You can think of it in terms of the actor model.
-pub(crate) struct EngineState {
+pub struct EngineState {
```

Comment:
I will undo this. At some point ran the migrations by importing schema core. No longer needed.